db/migrate/20170117110442_add_unique_constraint_to_host_uuid.rb

```diff
@@ -0,0 +1,9 @@
+class AddUniqueConstraintToHostUuid < ActiveRecord::Migration
+  def up
+    add_index(:hosts, [:uuid, :compute_resource_id], :unique => true)
```

Comment:
if there are any failures, this would fail, should we check it / notify it / skip it ?